x/poolmanager/create_pool.go

```diff
@@ -53,40 +52,78 @@ func (k Keeper) CreatePool(ctx sdk.Context, msg types.CreatePoolMsg) (uint64, er
 		return 0, err
 	}
 
+	// Send initial liquidity to the pool's address.
+	initialPoolLiquidity := msg.InitialLiquidity()
+	err = k.bankKeeper.SendCoins(ctx, sender, pool.GetAddress(), initialPoolLiquidity)
+	if err != nil {
+		return 0, err
+	}
```

Comment:
Is this just a no-op for CL pools?

Reply:
It is hardcoded to empty coins for CL message:
https://github.com/osmosis-labs/osmosis/blob/9b4b9f1c8c2e44b6425dc478c744de8eebb1c430/x/concentrated-liquidity/model/msgs.go#L105-L107